### PR TITLE
Update pddm to work with clang-format

### PR DESCRIPTION
**Pull request by dmaclach**
Add clang-format off/on directives around pddm expansions to simplify formatting other code.

### CODE REVIEW

File: objectivec/DevTools/pddm.py [hunk 1]
```diff
@@ -482,13 +482,14 @@ def lines(self):
...
           # option to the EXPAND to indicate if this should be done.)
-          result.extend([_GENERATED_CODE_LINE, ''])
+          result.extend([_GENERATED_CODE_LINE, '// clang-format off', ''])
           macro = line[directive_len:].strip()
           try:
             expand_result = self._macro_collection.Expand(macro)
             # Since expansions are line oriented, strip trailing whitespace
             # from the lines.
             lines = [x.rstrip() for x in expand_result.split('\n')]
+            lines.extend(['// clang-format on', ''])
```
Comment (thomasvl) [inline objectivec/DevTools/pddm.py]: The blank line being inserted here seems like a negative, is there a reason you added it?

Comment (dmaclach) [reply objectivec/DevTools/pddm.py]: Similar to the blank line in line 483 it just seemed to read better. Happy to remove if you prefer.

Comment (thomasvl) [reply objectivec/DevTools/pddm.py]: Go ahead and drop it.  It current sticks out with a blank before after, vs. merging with the block like what happens in that other case.

Comment (dmaclach) [reply objectivec/DevTools/pddm.py]: done

**Comment by dmaclach** [comment]
Updated with addition of clang-format to Makefile.am

**Comment by dmaclach** [comment]
Can you add kokoro:run again? I updated the tests appropriately

**Comment by thomasvl** [comment]
> Can you add kokoro:run again? I updated the tests appropriately

Done and looks good.

**Pull request merged**